app/assets/stylesheets/homepage.css.scss

```diff
@@ -71,14 +76,19 @@
     height: 550px;
     background: transparent image-url("homepage.png") no-repeat center -50px;
     background-size: cover;
-  box-shadow: 0px 30px 40px rgba(0,0,0,.5);
-
+  box-shadow: 0px 30px 40px ;
+  -moz-box-shadow:    0px 0px 40px 15px rgba(0, 0, 0, 0.6);
+  -webkit-box-shadow: 0px 0px 40px 15px rgba(0, 0, 0, 0.6);
+  box-shadow:         0px 0px 40px 15px rgba(0, 0, 0, 0.6);
```

Comment:
Property `box-shadow` already defined on line 79<br>Colon after property should be followed by one space<br>`0px` should be written without units as `0`